app_check/src/ios/common_ios.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIREBASE_APP_CHECK_SRC_IOS_COMMON_IOS_H_
+#define FIREBASE_APP_CHECK_SRC_IOS_COMMON_IOS_H_
+
+#import "FIRAppCheckToken.h"
+#include "firebase/app_check.h"
+
+namespace firebase {
+namespace app_check {
+
+AppCheckError AppCheckErrorFromNSError(NSError *_Nullable error);
```

Comment:
Any reason to not include these two in the app_check_ios.h header?

Reply:
I was thinking that app_check_ios.h contains internal versions of methods in the public AppCheck API whereas these are helpers to be used by providers and other app check implementation code.
But I would be ok with moving these into the app_check_ios header

Reply:
Actually, there is a relevant distinction:
These methods reference iOS concepts: NSError and FIRAppCheckToken
So the header that defines these methods cannot be included from a .cc file. And app_check.cc conditionally includes "app_check/src/ios/app_check_ios.h".

That said, since it is a separate header, I will make it a separate source file as well. And I will rename to util_ios and put it in the internal namespace to match app, database, and storage

Reply:
SGTM